City-of-Helsinki/drupal-helfi-etusivu
# UHF-6189 React app to Drupal

# [UHF-6189](https://helsinkisolutionoffice.atlassian.net/browse/UHF-6159)
Add the news archive app to Drupal.

## What was done
- Filled out info.yml in custom module `helfi_news_archive`
- Added block to render the app with
- Added installing archive app via composer.json
- Added post-install hooks to copy bundled js code custom module

## How to install
* Checkout branch. Run `make build`

## How to test
* Open up the site and go to [block layout management](https://helfi-etusivu.docker.so/fi/admin/structure/block)
* Add the block "News archive" somewhere on the site.
* View a page where the block should be visible.
* Check network tab from developer tools. You should see `main.597ef489.js` getting loaded.


## Designers review
